emacs-everywhere.el

```diff
@@ -460,9 +461,10 @@ Please go to 'System Preferences > Security & Privacy > Privacy > Accessibility'
                (pcase emacs-everywhere--display-server
                  (`(x11 . ,_) (emacs-everywhere--app-info-linux-x11))
                  (`(wayland . KDE) (emacs-everywhere--app-info-linux-kde))
+                 (`(wayland . Hyprland) (emacs-everywhere--app-info-linux-hyprland))
                  (_ (user-error "Unable to fetch app info with display server %S" emacs-everywhere--display-server)))))
     (make-emacs-everywhere-app
-     :id (string-to-number window-id)
+     :id window-id
```

Comment:
Hyprland window ids aren't numeric, might as well keep it as a string everywhere.